modules/framework-cats/shared/src/test/scala/DogFoodTests.scala

```diff
@@ -181,7 +181,7 @@ object DogFoodTests extends IOSuite {
         |  of
         |  multiline
         |  (failure)
-        |  assertion failed (modules/framework-cats/shared/src/test/scala/Meta.scala:34)
+        |  assertion failed (src/main/DogFoodTests.scala:5)
```

Comment:
These tests now display the implicit source location defined in the `Clue` object.